soaks/Dockerfile

```diff
@@ -19,7 +19,7 @@ FROM ghcr.io/blt/lading@sha256:1d2f7d3145c06df976bb9c06ad04c70de5b603598b3c4a85e
 # TARGET
 #
 FROM docker.io/debian:bullseye-slim@sha256:b0d53c872fd640c2af2608ba1e693cfc7dedea30abcd8f584b23d583ec6dadc7
-RUN apt-get update && apt-get dist-upgrade -y && apt-get -y install zlib1g ca-certificates
+RUN apt-get update && apt-get dist-upgrade -y && apt-get -y --no-install-recommends install zlib1g ca-certificates && rm -rf /var/lib/apt/lists/*
```

Comment:
If we could confirm that this still builds without the recommends I'd appreciate that.

Reply:
I tested against my `tilt/Dockerfile` which has the same command IIRC - but I'll confirm for these two soak files as well.

Reply:
This ended up failing for me locally - but I _suspect_ it's from my building on an M1.

Reply:
Actually the `rm -rf /var/lib/apt/lists/*` is not supposed to be needed. It's supposed to be done by the docker engine under the hood 🤔 But I like to keep it explicit.

Reply:
Interesting, it's one of the more common things I see around installing/updating packages.

Reply:
> Actually the `rm -rf /var/lib/apt/lists/*` is not supposed to be needed. It's supposed to be done by the docker engine under the hood 🤔 But I like to keep it explicit.

I don't think that is true? Do you have a source for that behaviour? I have never seen it.

You might not need it in multistage builds because intermediate stages are throw away. But I'm almost certain docker doesn't do cleaning by itself under the hood. How would it know what to clean ?

Reply:
I remember seeing it in the moby codebase, I'll look for it 😉